src/core/modules/SolarSystem.cpp

```diff
@@ -1266,10 +1266,10 @@ bool SolarSystem::loadPlanets(const QString& filePath)
 			const QString horizonMapName = ( hidden ? "" : englishName.toLower().append("_normals.png")); // no normal maps for invisible objects!
 
 			newP = PlanetP(new MinorPlanet(englishName,
-						    pd.value(secname+"/radius", 1.0).toDouble()/AU,
+						    pd.value(secname+"/radius", 0.0).toDouble()/AU,
 						    pd.value(secname+"/oblateness", 0.0).toDouble(),
 						    color, // halo color
-						    pd.value(secname+"/albedo", 0.25f).toFloat(),
+						    pd.value(secname+"/albedo", 0.15f).toFloat(),
```

Comment:
While this is in line with the estimate, is there a citable source for this estimate?

Reply:
Might as well default to 0 here. Added citation to updateEquatorialRadius.